index.html

```diff
@@ -331,245 +229,52 @@ <h2>Organizers</h2>
     </div>
     <div class="row aln-center gtr-150">
       <section class="special">
-        <img
-          src="./images/DES.png"
-          class="image profile-pic"
-          alt="Dev Edmonton Society"
-        />
+        <img src="./images/DES.png" class="image profile-pic" alt="Dev Edmonton Society" />
         <h3>Dev Edmonton Society</h3>
         <p><a href="https://github.com/devedmonton">GitHub</a></p>
       </section>
       <section class="special">
-        <img
-          src="./images/John.jpg"
-          class="image profile-pic"
-          alt="John Newton"
-        />
+        <img src="./images/John.jpg" class="image profile-pic" alt="John Newton" />
         <h3>John Newton</h3>
         <p>
           <a href="https://www.linkedin.com/in/johnnynewton/">LinkedIn</a>
         </p>
       </section>
       <section class="special">
-        <img
-          src="./images/Kevin.jpg"
-          class="image profile-pic"
-          alt="Kevin de Haan"
-        />
+        <img src="./images/Kevin.jpg" class="image profile-pic" alt="Kevin de Haan" />
         <h3>Kevin de Haan</h3>
         <p><a href="https://github.com/kdehaan">GitHub</a></p>
       </section>
       <section class="special">
-        <img
-          src="./images/Mandy.jfif"
-          class="image profile-pic"
-          alt="Mandy Meindersma"
-        />
+        <img src="./images/Mandy.jfif" class="image profile-pic" alt="Mandy Meindersma" />
         <h3>Mandy Meindersma</h3>
         <p><a href="https://github.com/MandyMeindersma">GitHub</a></p>
       </section>
       <section class="special">
-        <img
-          src="./images/Mark.png"
-          class="image profile-pic"
-          alt="Mark Bennett"
-        />
+        <img src="./images/Mark.png" class="image profile-pic" alt="Mark Bennett" />
         <h3>Mark Bennett</h3>
         <p><a href="https://github.com/markbennett">GitHub</a></p>
       </section>
       <section class="special">
-        <img
-          src="./images/Matthew.jpg"
-          class="image profile-pic"
-          alt="Matthew Mihok"
-        />
+        <img src="./images/Matthew.jpg" class="image profile-pic" alt="Matthew Mihok" />
```

Comment:
What's the difference on these lines @MandyMeindersma?

Reply:
oh oops my formatter is different, it just switched some indentation